apps/governance/src/routes/proposals/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { ProposalQuery } from './proposal/__generated__/Proposal';
+
+/**
+ * The default Proposal type needs extracting from the ProposalNode union type
+ * as lots of fields on the original type don't exist on BatchProposal. Eventually
+ * we will support BatchProposal but for now we don't
+ */
+export type Proposal = Extract<
```

Comment:
@edd @asiaznik  I've added this type to work around the fact that ProposalNode is now a union of Proposal and BatchProposal. Use this type for proposals to avoid type narrowing the union as right now we only have normal Proposals